.github/workflows/deploy-storybook.yml

```diff
@@ -0,0 +1,37 @@
+name: Deploy Storybook
+on:
+  push:
+    branches:
+      - main
+  workflow_dispatch:
+jobs:
+  deploy:
+    name: Deploy Storybook
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: actions/setup-node@v2-beta
+        with:
+          node-version: '14'
+
+      - name: Install dependencies
+        run: npm install
```

Comment:
It's not super important, since this won't run very often, but you might want to add some caching of the NPM dependencies so that this runs faster. There's a examples of this in the other workflows, but all you need to do is copy the following lines into this workflow file:

https://github.com/ShelterTechSF/sheltertech.org/blob/4fef794442e27e55770c5f2fb871f52925abf399/.github/workflows/deploy-staging.yml#L18-L23

And for what it's worth, these lines were copied verbatim from the documentation of [actions/cache](https://github.com/actions/cache), so we're not doing anything too fancy here: https://github.com/actions/cache/blob/v2.1.3/examples.md#macos-and-ubuntu

Reply:
Thanks for the suggestion -- added!